examples/ct/handlers.go

```diff
@@ -263,14 +263,14 @@ func parseBodyAsJSONChain(c LogContext, r *http.Request) (ct.AddChainRequest, er
 // TODO(Martin2112): Doesn't properly handle duplicate submissions yet but the backend
 // needs this to be implemented before we can do it here
 func addChainInternal(ctx context.Context, c LogContext, w http.ResponseWriter, r *http.Request, isPrecert bool) (int, error) {
-	var signerFn func(crypto.PrivateKeyManager, *x509.Certificate, *x509.Certificate, time.Time) (*ct.MerkleTreeLeaf, *ct.SignedCertificateTimestamp, error)
+	var makeLeafFn func(*x509.Certificate, *x509.Certificate, uint64) (*ct.MerkleTreeLeaf, error)
```

Comment:
I'd prefer to convert `Now()` to an RFC6962 timestamp once, and use that throughout.  Clarified comment to that effect.